src/software/world/robot_state.h

```diff
@@ -60,8 +62,15 @@ class RobotState
     AngularVelocity angularVelocity() const;
 
     /**
-     * Defines the equality operator for a RobotState. RobotStates are equal if their
-     * positions, velocities, orientation, and angular velocity are the same
+     * Returns the height of the robot in metres
+     *
+     * @return the height of the robot in metres
+     */
+    double height() const;
+
+    /**
+     * Defines the equality operator for a RobotState. RobotStates are equal if
+     * all their members are equal
      *
      * @param other The robot state to compare against for equality
      * @return True if the other robot state is equal to this robot state, and false
```

Comment:
Nit: Not part of your changes but the rest of the headers have whitespace between @param and @return.

Reply:
always happy to make things consistent. fixed.